src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -425,6 +425,48 @@ public class ConfigurationTests
             }
         }";
 
+        internal const string GRAPHQL_SCHEMA_WITH_CYCLE_ARRAY = @"
```

Comment:
why is this a configuration test? It is specific to cosmosdb, should be in one of the cosmos db tests.

Reply:
This file contains test specific to "Configuration". Database type doesn't matter. I can see cosmos related code in this file already.

Reply:
cosmos related code is there but that is testing the configuration. Your test is checking schema - which is only specific to cosmosdb.

Reply:
I considered passing schema is part of "configuring" cosmos DB.